R/function_Spectra_metrics.R

```diff
@@ -236,27 +236,38 @@ rtOverMSQuarters <- function(spectra, msLevel = 1L) {
 #' "The log ratio for the second to n-th quantile of TIC over the previous 
 #' quantile of TIC. For the boundary elements min/max are used." [PSI:QC]
 #' id: QC:4000058
+#'
+#' @note
+#'
+#' This function interprets the *quantiles* from the [PSI:QC] definition as
+#' *quartiles*, i.e. the 0, 25, 50, 75 and 100% quantiles are used.
 #' 
 #' @details
 #' is_a: QC:4000004 ! n-tuple
 #' is_a: QC:4000010 ! ID free
 #' is_a: QC:4000022 ! chromatogram metric
 #' is_a: QC:4000023 ! MS1 metric
-#' 
+#'
 #' The `log2` values are returned instead of the `log` values.
+#'
+#' *TIC changes* are interpreted as the cumulative sum (`cumsum`) of the
+#' spectras' TIC (with spectra ordered by retention time). Quartiles are then
+#' calculated on these. For *QC:4000057* the log2 ratio between the 25, 50, 75
+#' and 100% quartile to the 0% quartile is calculated. For *QC:4000058* 
+#' ratios between the 25/0, 50/25, 75/50 and 100/75% quartiles are calculated.
 #' 
 #' @param spectra `Spectra` object
+#' 
 #' @param relativeTo `character`
-#' @param MSLevel `numeric`
+#'
+#' @param msLevel `integer(1)`
 #' 
 #' @return `numeric(1)`
 #' 
 #' @author Thomas Naake, \email{thomasnaake@@googlemail.com}
 #' 
 #' @export
 #' 
-#' @importFrom ProtGenerics filterMsLevel tic ionCount
```

Comment:
I was only removing it because you imported these already in another function - but I'm OK with it. Maybe better to specifically add the import in each function you use them.